// Validating benchmarks:
// ***** BenchmarkRunner: Start   *****
// ***** Found 1 benchmark(s) in total *****
// ***** Building 1 exe(s) in Parallel: Start   *****
// start dotnet  restore /p:UseSharedCompilation=false /p:BuildInParallel=false /m:1 /p:Deterministic=true /p:Optimize=true /p:IntermediateOutputPath="C:\Dev\TestDynamo\tests\TestDynamo.Tests\bin\Release\net8.0\949612fe-9253-4d23-9366-f1876910e6dc\obj\Release\net8.0/" /p:OutDir="C:\Dev\TestDynamo\tests\TestDynamo.Tests\bin\Release\net8.0\949612fe-9253-4d23-9366-f1876910e6dc\bin\Release\net8.0/" /p:OutputPath="C:\Dev\TestDynamo\tests\TestDynamo.Tests\bin\Release\net8.0\949612fe-9253-4d23-9366-f1876910e6dc\bin\Release\net8.0/" in C:\Dev\TestDynamo\tests\TestDynamo.Tests\bin\Release\net8.0\949612fe-9253-4d23-9366-f1876910e6dc
// command took 1.37 sec and exited with 0
// start dotnet  build -c Release --no-restore /p:UseSharedCompilation=false /p:BuildInParallel=false /m:1 /p:Deterministic=true /p:Optimize=true /p:IntermediateOutputPath="C:\Dev\TestDynamo\tests\TestDynamo.Tests\bin\Release\net8.0\949612fe-9253-4d23-9366-f1876910e6dc\obj\Release\net8.0/" /p:OutDir="C:\Dev\TestDynamo\tests\TestDynamo.Tests\bin\Release\net8.0\949612fe-9253-4d23-9366-f1876910e6dc\bin\Release\net8.0/" /p:OutputPath="C:\Dev\TestDynamo\tests\TestDynamo.Tests\bin\Release\net8.0\949612fe-9253-4d23-9366-f1876910e6dc\bin\Release\net8.0/" --output "C:\Dev\TestDynamo\tests\TestDynamo.Tests\bin\Release\net8.0\949612fe-9253-4d23-9366-f1876910e6dc\bin\Release\net8.0/" in C:\Dev\TestDynamo\tests\TestDynamo.Tests\bin\Release\net8.0\949612fe-9253-4d23-9366-f1876910e6dc
// command took 34.27 sec and exited with 0
// ***** Done, took 00:00:35 (35.71 sec)   *****
// Found 1 benchmarks:
//   Runner.Run: DefaultJob

Setup power plan (GUID: 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c FriendlyName: High performance)
// **************************
// Benchmark: Runner.Run: DefaultJob
// *** Execute ***
// Launch: 1 / 1
// Execute: dotnet 949612fe-9253-4d23-9366-f1876910e6dc.dll --anonymousPipes 1912 1844 --benchmarkName Program+Program+Runner.Run --job Default --benchmarkId 0 in C:\Dev\TestDynamo\tests\TestDynamo.Tests\bin\Release\net8.0\949612fe-9253-4d23-9366-f1876910e6dc\bin\Release\net8.0
// BeforeAnythingElse
Seed 1452905967

// Benchmark Process Environment Information:
// BenchmarkDotNet v0.14.0
// Runtime=.NET 8.0.11 (8.0.1124.51707), X64 RyuJIT AVX2
// GC=Concurrent Workstation
// HardwareIntrinsics=AVX2,AES,BMI1,BMI2,FMA,LZCNT,PCLMUL,POPCNT VectorSize=256
// Job: DefaultJob

OverheadJitting  1: 1 op, 169600.00 ns, 169.6000 us/op

System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.ArgumentException: File not found: C:\Dev\TestDynamo\TestDynamo.Tests\bin\Debug\net8.0\TestDynamo.Tests.dll (Parameter 'assemblyFileName')
   at Guard.ArgumentValid(String argName, Boolean test, String messageFormat, Object[] args) in /_/src/common/Guard.cs:line 41
   at Guard.FileExists(String argName, String fileName) in /_/src/common/Guard.cs:line 50
   at Xunit.XunitFrontController..ctor(AppDomainSupport appDomainSupport, String assemblyFileName, String configFileName, Boolean shadowCopy, String shadowCopyFolder, ISourceInformationProvider sourceInformationProvider, IMessageSink diagnosticMessageSink) in /_/src/xunit.runner.utility/Frameworks/XunitFrontController.cs:line 64
   at Xunit.Runners.AssemblyRunner..ctor(AppDomainSupport appDomainSupport, String assemblyFileName, String configFileName, Boolean shadowCopy, String shadowCopyFolder) in /_/src/xunit.runner.utility/Runners/AssemblyRunner.cs:line 57
   at Xunit.Runners.AssemblyRunner.WithoutAppDomain(String assemblyFileName) in /_/src/xunit.runner.utility/Runners/AssemblyRunner.cs:line 294
   at Program.Program.Runner.Run() in C:\Dev\TestDynamo\tests\TestDynamo.Tests\Program.fs:line 128
   at BenchmarkDotNet.Autogenerated.Runnable_0.WorkloadActionNoUnroll(Int64 invokeCount) in C:\Dev\TestDynamo\tests\TestDynamo.Tests\bin\Release\net8.0\949612fe-9253-4d23-9366-f1876910e6dc\949612fe-9253-4d23-9366-f1876910e6dc.notcs:line 311
   at BenchmarkDotNet.Engines.Engine.Measure(Action`1 action, Int64 invokeCount)
   at BenchmarkDotNet.Engines.Engine.RunIteration(IterationData data)
   at BenchmarkDotNet.Engines.EngineFactory.Jit(Engine engine, Int32 jitIndex, Int32 invokeCount, Int32 unrollFactor)
   at BenchmarkDotNet.Engines.EngineFactory.CreateReadyToRun(EngineParameters engineParameters)
   at BenchmarkDotNet.Autogenerated.Runnable_0.Run(IHost host, String benchmarkName) in C:\Dev\TestDynamo\tests\TestDynamo.Tests\bin\Release\net8.0\949612fe-9253-4d23-9366-f1876910e6dc\949612fe-9253-4d23-9366-f1876910e6dc.notcs:line 176
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodBaseInvoker.InvokeDirectByRefWithFewArgs(Object obj, Span`1 copyOfArgs, BindingFlags invokeAttr)
   --- End of inner exception stack trace ---
   at System.Reflection.MethodBaseInvoker.InvokeDirectByRefWithFewArgs(Object obj, Span`1 copyOfArgs, BindingFlags invokeAttr)
   at System.Reflection.MethodBaseInvoker.InvokeWithFewArgs(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at BenchmarkDotNet.Autogenerated.UniqueProgramName.AfterAssemblyLoadingAttached(String[] args) in C:\Dev\TestDynamo\tests\TestDynamo.Tests\bin\Release\net8.0\949612fe-9253-4d23-9366-f1876910e6dc\949612fe-9253-4d23-9366-f1876910e6dc.notcs:line 57
// AfterAll
No Workload Results were obtained from the run.
// Benchmark Process 17232 has exited with code -1.

// ** Remained 0 (0.0%) benchmark(s) to run. Estimated finish 2024-11-18 13:31 (0h 0m from now) **
Successfully reverted power plan (GUID: 381b4222-f694-41f0-9685-ff5bb260df2e FriendlyName: Balanced)
// ***** BenchmarkRunner: Finish  *****

// * Export *
  BenchmarkDotNet.Artifacts\results\Program.Program.Runner-report.csv
  BenchmarkDotNet.Artifacts\results\Program.Program.Runner-report-github.md
  BenchmarkDotNet.Artifacts\results\Program.Program.Runner-report.html

// * Detailed results *
Runner.Run: DefaultJob
Runtime = .NET 8.0.11 (8.0.1124.51707), X64 RyuJIT AVX2; GC = Concurrent Workstation
There are not any results runs

// * Summary *

BenchmarkDotNet v0.14.0, Windows 11 (10.0.22631.4460/23H2/2023Update/SunValley3)
AMD Ryzen 7 5800H with Radeon Graphics, 1 CPU, 16 logical and 8 physical cores
.NET SDK 8.0.307
  [Host]     : .NET 8.0.11 (8.0.1124.51707), X64 RyuJIT AVX2 DEBUG
  DefaultJob : .NET 8.0.11 (8.0.1124.51707), X64 RyuJIT AVX2


| Method | Mean | Error |
|------- |-----:|------:|
| Run    |   NA |    NA |

Benchmarks with issues:
  Runner.Run: DefaultJob

// * Warnings *
Environment
  Summary -> Detected error exit code from one of the benchmarks. It might be caused by following antivirus software:
        - Windows Defender (windowsdefender://)
Use InProcessEmitToolchain or InProcessNoEmitToolchain to avoid new process creation.


// * Legends *
  Mean  : Arithmetic mean of all measurements
  Error : Half of 99.9% confidence interval
  1 ns  : 1 Nanosecond (0.000000001 sec)

// * Diagnostic Output - MemoryDiagnoser *

// * Diagnostic Output - ExceptionDiagnoser *


// ***** BenchmarkRunner: End *****
Run time: 00:00:06 (6.69 sec), executed benchmarks: 1

Global total time: 00:00:42 (42.61 sec), executed benchmarks: 1
// * Artifacts cleanup *
Artifacts cleanup is finished
